Templates/BaseGame/game/core/gui/scripts/canvas.tscript

```diff
@@ -202,27 +209,34 @@ function GuiCanvas::checkCanvasRes(%this, %mode, %deviceId, %deviceMode, %startu
    return false;
 }
 
-// Find the best video mode setting for the device and display mode
+// Find the best video mode setting for the device and display mode.
+// "Best" is the largest resolution that will fit at highest refresh rate.
```

Comment:
Thanks for improving the comments! :)

I believe TorqueScript supports doxygen-ish comments if you do three slashes, but it's not really a heavily-used feature